models/face_image_quality_assessment_ediffiqa/demo.py

```diff
@@ -0,0 +1,159 @@
+# This file is part of OpenCV Zoo project.
+# It is subject to the license terms in the LICENSE file found in the same directory.
+
+
+import sys
+import argparse
+
+import numpy as np
+import cv2 as cv
+
+from skimage import transform as trans
```

Comment:
Could it be replaced with opencv functions?

Basically we use `opencv` for any image operations and `numpy` for array operations.

Reply:
The skimage function call was replaced using the appropriate OpenCV function call.  

`tfm, _ = cv.estimateAffinePartial2D(src_pts, ref_pts, method=cv.LMEDS)`